.github/actions/release/action.yml

```diff
@@ -13,55 +13,182 @@ inputs:
   jira_project_id:
     description: "Jira project id to create release"
     required: true
-  jira_base_url:
-    description: "Jira base url"
+  jira_webhook_url:
+    description: "Jira release webhook"
     required: true
 
 runs:
   using: "composite"
   steps:
-    - name: Publish RPMS to Repositories
+    - name: Checkout sources
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+    - name: Get released versions for components
       run: |
-        NEW_VERSION=""
-        MAJOR_VERSION=$(echo $GITHUB_REF_NAME | grep -oP '([0-9]{2}\.[0-9]{2})')
-        echo "Major version: $MAJOR_VERSION"
-        HOTFIX=$(echo $GITHUB_REF_NAME | grep -oP '(hotfix|)')
-        echo "Hotfix: $HOTFIX"
-        RELEASE_ID=$(git log -1 --pretty=%B | grep -oP '(#[0-9]{4,}#)' | grep -oP '([0-9]+)')
-        echo "Release Id: $RELEASE_ID"
+        set -eu
+
+        COMPONENTS_OSS=("centreon-awie" "centreon-dsm" "centreon-gorgone" "centreon-ha" "centreon-open-tickets" "centreon-web")
+        COMPONENTS_MODULES=("centreon-anomaly-detection" "centreon-autodiscovery" "centreon-bam" "centreon-cloud-business-extensions" "centreon-cloud-extensions" "centreon-it-edition-extensions" "centreon-license-manager" "centreon-map" "centreon-mbi" "centreon-pp-manager")
+        COMPONENTS_COLLECT=("centreon-collect")
```

Comment:
useless

Reply:
I disagree on this, since the workflow is aiming to be unified and applicable to any of OSS COLLECT or MODULES.
the workflows have no way to identify components unless we declare them explicitely (relying on existing workflow files is not a valid way to identify them)

Reply:
it is not the case. the variable is not managed by the reusable workflow.
in this action, this is out of context and useless